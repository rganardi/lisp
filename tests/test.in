;try to segfault
(define 0 (quote a))
(define add1
  (lambda (x)
    (lambda () x)))
(define 2 (add1 (add1 0)))
2
(2)
;((2))
(add1 0)
;((add1 0))
