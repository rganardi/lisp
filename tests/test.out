parse_eval: unspecified
parse_eval: unspecified
parse_eval: unspecified
(lambda () (lambda () a))
(lambda () a)
(lambda () a)
